redditAPI.py

```diff
@@ -36,6 +38,15 @@ def get_access_token():
         print(f"Error: {e}")
         return None
 
+
+def clean_title(title):
+    # Remove non-ASCII characters
```

Comment:
remove all unicode characters instead of just non-ASCII characters.

Reply:
Done, taken care of!